charts/hivemq-swarm/templates/deployment_agents.yaml

```
@@ -0,0 +1,90 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "hivemq-swarm.fullname" . }}-agent
```

Since we provide the override function - I suggest to use it in all templates
```suggestion
  name: {{ include "hivemq-swarm.fullname" . }}-agent
  namespace: {{ template "hivemq-swarm.namespace" . }}
```